identity-composer.html

```diff
@@ -1,70 +1,56 @@
-      <span ng-if="!choice.selected"
-        class="fa fa-question-circle"></span>
-      <span ng-if="choice.selected"
-        class="fa fa-check-circle text-success"></span>
-      {{choice.label}}<span ng-if="choice.optional"> (optional)</span>
-    </button>
-    <button
-      type="button" class="btn btn-default"
-      ng-disabled="!composed"
-      ng-click="done()">Done</button>
+<div ng-show="page == 'edit'" class="row row-centered edit-header">
+  <div class="col-xs-2">
+    <a class="text-center" ng-click="back()"><span style="margin-top: 25px;" class="glyphicon glyphicon-chevron-left"/>Back</a>
   </div>
-  <div ng-repeat="(property, choice) in choices track by property">
-    <div ng-show="choice.show">
-      <h4>Select a credential for <strong>{{choice.label}}</strong>:</h4>
-      <ul class="list-unstyled">
-        <li class="br-selectable well"
-          ng-class="{'br-selected': choice.selected === option.credential}"
-          ng-repeat="option in choice.options track by $index"
-          ng-click="choice.selected = option.credential">
-          <!-- TODO: replace with br-credential-thumbnail -->
-          <div class="section">
-            <h3 class="headline" style="margin-top: 5px">
-              {{option.credential.name || 'Credential'}}
-              <span class="btn-group pull-right">
-                <button type="button"
-                  class="btn btn-default"
-                  ng-click="$event.stopPropagation(); showCredential(option)">
-                  <i class="fa fa-trophy fa-lg"></i>
-                </button>
-              </span>
-            </h3>
-          </div>
-          <br-credential
-            br-credential="option.credential"
-            br-show-actions="false"></br-credential>
-        </li>
-        <li class="well"
-          ng-if="choice.options.length == 0">
-          <strong>No credentials found.</strong>
-        </li>
-      </ul>
+  <div class="col-xs-8">
+    <h2 class="text-center">Edit Selection</h2>
+  </div>
+</div>
+<div ng-show="page == 'front'" class="row text-center front-page">
+  <div class="row">
+    <h4 class="">A site is requesting the following credentials:</h4>
+  </div>
+  <div class="credential-container">
+    <p ng-repeat="credential in selectedCredentials">{{credential.name}} <a class="glyphicon glyphicon-info-sign" tooltip-html-unsafe="{{htmlClaimsForCredential(credential)}}" tooltip-placement="right"></a> <span ng-if="fulfillsExcessClaims(credential).length > 0" style="color: #FFBB36;" class="glyphicon glyphicon-alert" tooltip-html-unsafe="{{htmlExcessClaimsForCredential(credential)}}" tooltip-placement="right"></span></p>
+  </div>
+  <div class="row" style="margin-bottom: 30px;">
+    <a style="margin-bottom: 20px;" ng-click="page = 'edit'">
+      Edit which credentials you're sending
+    </a>
+    <h5 tooltip-html-unsafe="{{htmlClaims()}}" ng-if="selectedCredentials.length == 1">Why does this site need this credential?</h5>
```

Comment:
Would like to change these over to popovers or something else, but the version of angular-bootstrap we are using is 0.12.1 which doesn't support it. The latest version that does that doesn't look like it will horribly break everything is 0.13.4, can we migrate over at some point?